tests/__init__.py

```diff
@@ -14,3 +15,7 @@ def read_file(self, filename):
         if self.data_dir:
             with open(os.path.join(self.data_dir, filename)) as fp:
                 return fp.read()
+
+    def compare_queries(self, q1,q2):
+        # It normalizes the queries in the same format and suppresses format differences (i.e. spaces, tables, \n)
+        return print_ast(parse(q1)) == print_ast(parse(q2))
```

Comment:
this is a great trick, thanks!

I'm going to make one change, this method will just return `True` or `False`, and then we will have an assert in the test that checks this. If this check fails, pytest will give us no information about which bits are different. The better solution would be to have in this method `assert print_ast(parse(q1)) == print_ast(parse(q2))`, and then just call the method from each test without using assert.

I'll make these changes and then merge it.